quarkus-extension/engine/deployment/src/test/java/org/camunda/bpm/quarkus/engine/test/ManagedJobExecutorTest.java

```diff
@@ -129,6 +128,7 @@ public void shouldNotReuseManagedExecutor() {
   @Deployment
   public void shouldExecuteJob() {
     // given
+    processEngineConfiguration.getJobExecutor().shutdown();
```

Comment:
:thought_balloon: Does it make sense to move this to the `#setUp()` to avoid similar issues in case of future tests? I think we don't need the job executor running in the other tests, we only check if it's active via the process engine configuration which is different thing.

Reply:
If we move it to the `setup()`, there might be tests that are affected by the stopping of the scheduler e.g `shouldActivateJobExecutorByDefault` which tests that the scheduler is activated by default 🤔

I had similar thoughts of turning off the respective property `quarkus.camunda.job-executor.activate=false` for every test but didn't favour it in the end due to the same reason.